### PR TITLE
Milestone: Created  detailed pop window

### PR DESCRIPTION
**Pull request by Roseokpe**
For this milestone of my portfolio website, I implement a popup window that includes the project details in both desktop and mobile.

### CODE REVIEW

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @Roseokpe, 👋  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ 

You have submitted a project, that is passing the linters check, this is what you can do:

Captura de pantalla 2022-05-20 a la(s) 10 36 55 p  m

`Expected new line` means you need to add an empty line after the mentioned line.

`Unexpected whitespace` means you have a whitespace at the end of some line, for example, if you have `x=0;  `  you need to delete the whitespace at the end `x=o;` this whitespace is invisible. Use backspace to make sure you don't have whitespace.

You can use  `stylelint "**/*.scss" --syntax scss --fix` to fix most of the errors automatically 
 
Captura de pantalla 2022-05-20 a la(s) 10 36 41 p  m

`Unexpected function` means you need to use arrow functions.
`Expected indentation` means you need to add or remove whitespaces at the beginning of the line.
'Trailing spaces` means you have whitespace at the end of the line, remove them.

You can use  `npx eslint --fix .` to fix most of the errors automatically 

You can google each error to see how to fix it, you are almost there!

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -21,4 +22,28 @@ function hideMenu() {
...
+document.querySelector('.menu-list').addEventListener('click', hideMenu);
+
+// popup
+const modal = document.getElementById('myModal');
+const raiseModal = document.querySelectorAll('.raise-modal');
+raiseModal.forEach((trigger) => {
+  trigger.addEventListener('click', () => {
+    modal.style.display = 'block';
+  });
+});
```
Comment (AmaduKamara) [inline index.js]: I noticed that you created a static modal in your HTML that you're fetching and displaying in this function.

- [ ] Kindly create an array of projects holding objects of the different projects. It's required that you have that array in a `.js` file, loop through that array, and populate each modal upon clicking with its details in the object. Kindly work on that. 👍 

### Hint:
```mdx
const projects = [{
    name: "Name of the project",
    description: "description of the project",
    image: "link_to_image.jpg",
    technologies: ["HTML", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
``` 
- Please use that format to store the popup details for every project and pass it to the modal when clicked. 👍

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @Roseokpe,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Correct use of Github workflow
✅ No linter error

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly implement the array feature in order for your project to be marked as completed.
**It was required in your project brief.**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by m05e5 — APPROVED** [review]
# Hi @Roseokpe , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**